components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -248,7 +248,12 @@ tweaking their $case/namelist_scream.xml file.
     ${DIN_LOC_ROOT}/atm/scream/tables/vm_table_vals.dat8,
     ${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30_20220428.nc,
     ${DIN_LOC_ROOT}/atm/scream/init/map_ne30_to_ne4_mono_20220502.nc,
-    ${DIN_LOC_ROOT}/atm/scream/init/map_ne30np4_to_ne120np4_mono_20220502.nc
+    <!-- ${DIN_LOC_ROOT}/atm/scream/init/map_ne30np4_to_ne120np4_mono_20220502.nc -->
```

Comment:
Do we just not need these files to be downloaded anymore?

Reply:
I noticed that we were downloading remap files for all resolutions in order to run at any particular resolution, but weren't downloading any initial condition at all. I decided to set us up to download all data for all resolutions, then commented out the most biggest resolutions because I worry that someone might want to run ne4 and ned up waiting hours for ne512 IC to download... What I was hoping for was to have all the files show up in namelist_scream.xml, but to have ne120 and ne512 be commented out. Then folks can uncomment/download them as needed. Of course, it would be better to only download the files we need, but that's a v1.1 task.

Reply:
I'm okay with that.  super important to make sure that every item is trailed by a comma.  Maybe we should make sure that the last item in the entry is something we will for sure keep uncommented in the list and have it be the only thing without a comma.  That comma is subtle and tricky.